src/main/java/htsjdk/variant/variantcontext/VariantContext.java

```diff
@@ -787,10 +788,26 @@ public boolean emptyID() {
         return ! hasID();
     }
 
+    /**
+     * @return the ID field of the record, this be {@linkplain VCFConstants#EMPTY_ID_FIELD}, a single ID, or a semi-colon
```

Comment:
Not sure why you'd want to use `linkplain` here as this text is code, so `@link` seems more appropriate.

Reply:
👍